db/organizations/twitter_counter.eno

```diff
@@ -1,9 +1,9 @@
-name: Twitter Counter
+name: X Counter
```

Comment:
I would suggest to leave the old name here (Twitter Counter). Looks like it is discontinued (since 2018):
https://en.wikipedia.org/wiki/TwitterCounter

Maybe we can adjust the text to something like:
"Twitter Counter was a is a third party application for Twitter (now X) that provides usage statistics."

And mention that it ceased operations on November 5, 2018.